### PR TITLE
Seyunchung

### PR DESCRIPTION
**Pull request by SeyunChung**
I don't know why, but I just added guest review section and chat section for any Q&A

### CODE REVIEW

**Review by ryanmphill — CHANGES_REQUESTED** [review]
Looks good, but maybe we should wrap all of this in a new `<section>` container!

**Comment by SeyunChung** [comment]
do you like to put these under my name `<section class="Seyun">_________</section> `section?

**Comment by ryanmphill** [comment]
@SeyunChung Sure! Sounds good to me

File: index.html [hunk 1]
```diff
@@ -14,13 +14,7 @@
 <section class="Jonathan"></section>
 <section class="Logan"></section>
 <section class="Ryan"></section>
-<section class="Seyun"></section>
```
Comment (ryanmphill) [inline index.html:17]: This is where you would want to paste all of your code

**Comment by SeyunChung** [comment]
I think I fixed it, please review it for me. Thanks

**Review by ryanmphill — APPROVED** [review]
Looks good!

**Comment by SeyunChung** [comment]
which button should I click? just first "create a merge commit"? @ryanmphill

**Comment by ryanmphill** [comment]
I think you got it! Once it is approved you can merge here in the pull request on the github website

**Pull request merged**